utils/generate-commands-json.py

```diff
@@ -88,10 +90,6 @@ def convert_command_to_objects(container, cmd):
 
     obj[key] = {
         **obj[key],
-        'arity': cmd[1],
-        'first': cmd[3],
-        'last': cmd[4],
-        'step': cmd[5],
```

Comment:
I think that since they're part of COMMAND output, they should be in the json, whoever doesn't need them, won't use them

Reply:
Arity is meaningful, but we want to move people away from first/last/step and towards keys_specs (or something even better). No?

Reply:
i think maybe some existing users of (the old) json file, will want to get the new one that includes all commands, but they still depend on the primitive `first, last, step` thing, so we still wanna provide that.
maybe as a "legacy" entry (since the json layout is changing anyway)?

i.e. i was thinking we wanna include all the info from COMMAND command.
but maybe you're right, and we should exclude the things we only kept there for backwards compatibility.
in which case do we also wanna trim the `movablekeys` flag?

@guybe7 WDYT?

Reply:
yes, let's drop all deprecated/obsolete info

we want people to slowly move towards key-specs